AUTHORS.txt

```diff
@@ -631,3 +631,4 @@ Zvezdan Petkovic
 Łukasz Langa
 Семён Марьясин
 ‮rekcäH nitraM‮
+Andrew Meleka
```

Comment:
This file is auto-generated. Please don't edit it.

Reply:
Sorry :(